src/lsptoolshost/server/languageServerEvents.ts

```diff
@@ -18,12 +18,25 @@ export interface ServerStateChangeEvent {
     workspaceLabel: string;
 }
 
+export interface ProjectsRestoredEvent {
```

Comment:
We probably want to include more info in these events such as the list of projects that were restored or whose load started/completed.